### PR TITLE
Have different message when a closed PR is merged or not

### LINKED ISSUE #25: Messages does not distinguish if a closed PR was merged or not.
**Describe the bug**
If a PR is closed without merging the added message to task says "merged":
> toffentoffen merged a PR: conversion reports refactor for stages

**To Reproduce**
1. Close a PR without merging.

**Expected behavior**
Message should be:
toffentoffen closed a PR without merging: conversion reports refactor for stages

**Additional context**

### PR DESCRIPTION
**Pull request by toffentoffen**
Fixes #25 
Have different messages for when a closed pr is merged or not.

### CODE REVIEW

File: src/teamwork.sh [hunk 1]
```diff
@@ -47,11 +47,19 @@ teamwork::pull_request_closed() {
   local -r user=$(github::get_sender_user)
   local -r pr_url=$(github::get_pr_url)
   local -r pr_title=$(github::get_pr_title)
+  local -r pr_merged=$(github::get_pr_merged)
```
Comment (miguelbemartin) [inline src/teamwork.sh:50]: You need to create this function to fetch this field from the JSON file. This function doesn't exist. 

You can find in the `github.sh` file all the helpers to get information from Github, and by using `jq` library you can easily process the JSON file.

**Review by miguelbemartin — CHANGES_REQUESTED** [review]
Nice one, Marc. It looks good overall, just a small missing piece of code.

File: src/teamwork.sh (hunk 1 above)
Comment (toffentoffen) [reply src/teamwork.sh:50]: oh maaaaan, I hate vscode, always forget to save the file.
I was there hanging unsaved.

**Comment by miguelbemartin** [comment]
Nice one, Marc 👍

**Review by miguelbemartin — APPROVED** [review]

**Pull request merged**